content/_includes/footer.html

```diff
@@ -0,0 +1,10 @@
+<footer class="{{ page.class }} site-footer h-card">
+<div class="container">
+  <div class="row">
+    <div class="col-xs-12 footer-text">
+    	<p>Copyright © 2019 <span>•</span> <a href="{{ site.url}}{{ site.baseurl }}">{{ site.title }}</a> <span>•</span> <a href="{{ "terms-of-service/" | absolute_url }}">Terms of Service</a> <span>•</span> <a href="{{"privacy-policy" | absolute_url }}">Privacy Policy</a>.</p>
```

Comment:
```suggestion
    	<p>Copyright © 2014-2020 <span>•</span> <a href="{{ site.url}}{{ site.baseurl }}">{{ site.title }}</a> <span>•</span> <a href="{{ "terms-of-service/" | absolute_url }}">Terms of Service</a> <span>•</span> <a href="{{"privacy-policy" | absolute_url }}">Privacy Policy</a>.</p>
```

Reply:
Needs to be 2014-2021 now

Reply:
Says 2019?

Reply:
This was from Dec 20. I updated this earlier today, and marked it as resolved. It currently reads: 
`<p>Copyright © 2014-{{ site.time | date: '%Y' }} <span>•</span> <a href="{{ site.url}}{{ site.baseurl }}">{{ site.title }}</a> <span>•</span> <a href="{{"privacy" | absolute_url }}">Privacy Policy</a>.</p>` - which reflects on the staging site.

Going to mark as resolved.